src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -93,7 +108,13 @@ public Object instanceMain(final String[] args, final List<String> packageList,
         // we can lop off the first two arguments but it requires an array copy or alternatively we could update CLP to remove them
         // in the constructor do the former in this implementation.
         final String[] mainArgs = Arrays.copyOfRange(args, 1, args.length);
-        return program.instanceMain(mainArgs);
+        try {
+            return program.instanceMain(mainArgs);
+        } catch (final CommandLineException e){
+            //Print usage in the case of a CommandLineException
+            System.err.println(program.getUsage());
```

Comment:
If it is complicated, I think that it could be maintained here. I'm just thinking in putative usages in the future for me or other framework users...